stockflux-securities-master/src/components/InputForm.js

```diff
@@ -133,6 +150,18 @@ const InputForm = ({ match }) => {
           </div>
         </form>
       )}
+      <ul>
+        {sendingMessages.map(message => {
```

Comment:
Here you're creating a list of messages and creating a `Message` component instance for each one. Consider making your `Message` component accept an array of messages and have it create a *single* message list, with an `<li>` for each message. Then you can just pass the messages into the component and let the component deal with rendering it.

Reply:
Done now